packages/datadog-plugin-hapi/test/index.spec.js

```diff
@@ -101,7 +101,8 @@ describe('Plugin', () => {
             expect(traces[0][0].meta).to.have.property('span.kind', 'server')
             expect(traces[0][0].meta).to.have.property('http.url', `http://localhost:${port}/user/123`)
             expect(traces[0][0].meta).to.have.property('http.method', 'GET')
-            expect(traces[0][0].meta).to.have.property('http.status_code', '200')
+            expect(traces[0][0].meta).to.have.property('http.status_code')
+            expect(Number(traces[0][0].meta['http.status_code'])).to.be.within(200, 299)
```

Comment:
Why is this change here?

Reply:
`hapi` switched from `200` to `204` when the response is empty. For the purpose of the test, we don't care as long as it's a 2xx.